starter/script.js

```diff
@@ -1 +1,233 @@
 // put code here!
+window.alert("Welcome to the Magic Kingdom! The Happiest Place on Earth!" +
+"\nLet's take a tour of the park together!");
+
+window.alert("Our first stop is Main Street USA. The street is bustling with shops and entertainment. You can see the Dapper Dans singing barbershop quartet."
++ "\nFun fact: Each business name in the shop windows refers to a person who had some part in the building of Disney World.");
+
+var time = window.prompt("How many minutes should we stay to listen to the songs?");
+
+if(time > 0)
+{
+  window.alert("You listen to " + time + " minute(s) of fantastic singing.");
+}
+else {
+  window.alert("Hmm, I guess the Dans are not your thing...");
+}
+
+
+
+window.alert("Let's continue forward to the Hub!");
+
+while(response != "Q" && response != "q")
+{
+  window.alert("In the Hub, you can see Cinderella's castle in front of you and several paths that go in different directions." +
+  "\nFrom here, you can go anywhere in the park!");
+
+  var response = window.prompt("Where do you want to go?"
+  + "\n Right to Tomorrowland (R)"
+  + "\n Straight to Fantasyland (S)"
+  + "\n Left to Adventureland (L)"
+  + "\n Go home (Q)");
+
+  if(response == "R" || response == "r")
```

Comment:
might be an example where a `switch` [statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) could make code a little more organized